src/security-events/EventMetadata.ts

```diff
@@ -0,0 +1,97 @@
+import {clearUndefinedProperties} from "../Util";
+
+export interface EventMetadataJson {
+    tenantId: string;
+    timestampMillis: number;
+    iclFields: {
+        requestId?: string;
+        event?: string;
+        sourceIp: string | undefined;
+        objectId: string | undefined;
+        requestingId: string;
```

Comment:
Might be helpful to maintain consistency and call this `requestingUserOrServiceId`

Reply:
Good catch.

Reply:
Oh sorry, thought this was on the constructor. No, that would be inconsistent. The public interface is the only place it's called that, and it's the same as the Java side. It'd require many more changes, and not for any benefit to us.